packages/client/src/constants/Env.ts

```diff
@@ -2,4 +2,7 @@ export const env = {
   HOST_API: 'https://ya-praktikum.tech/api/v2',
   HOST_RESOURCES: 'https://ya-praktikum.tech/api/v2/resources',
   HOST_WS: 'wss://ya-praktikum.tech/ws',
+  REDIRECT_URI: 'http://localhost:5000',
```

Comment:
```suggestion
  REDIRECT_URI: 'http://localhost:5000/',
```

пришел ответ, что теперь заработает oauth)